os-eap-launch/added/launch/datasource-common.sh

```diff
@@ -197,7 +197,11 @@ function generate_datasource_common() {
 
   if [ -n "$TIMER_SERVICE_DATA_STORE" -a "$TIMER_SERVICE_DATA_STORE" = "${service_name}" ]; then
     inject_timer_service ${pool_name}_ds
-    inject_datastore $pool_name $jndi_name $driver
+    local refresh_interval
+    if [[ "$JBOSS_EAP_VERSION" == "7."* ]]; then
```

Comment:
Unfortunately, this will not work across all EAP based products as JBOSS_EAP_VERSION may not defined (as a matter of fact, it's only defined for EAP 6.4 and EAP 7.x images).  This will need to be broken out into the EAP 7 specific datasource configuration script.

Reply:
Good catch Rob, thanks, I'll rework it and update this pr for your review.

Reply:
@spolti Status? This needs to get updated per @rcernich 's note above before RHPAM-181 can move forward. Thanks!

Reply:
@errantepiphany I am testing this change with eap images and also with rhpam kieserver image.
Probably by tomorrow morning I have this pr updated.